uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/leave-request-popup-controller.js

```diff
@@ -443,6 +512,16 @@ define([
       }
 
       /**
+       * Converts given date to javascript date as expected by uib-datepicker
+       *
+       * @param {Date} date
+       * @return {Date} Javascript date
```

Comment:
`Converts given date to javascript date as expected by uib-datepicker` is the purpose of the function so its taking a date in server format and reformatting it to a javascript date as uibdatepicker does not accepts moment date